examples/complete/README.md

```diff
@@ -0,0 +1,48 @@
+# atlas-basic - complete
+
+This example shows how the atlas-basic module can be used to create a MongoDB Atlas project, an AWS cluster for the project, several IP addresses and CIDR blocks for the project, and several database users. 
+
+All the input variables' values are given by the developer. 
```

Comment:
This is more of an implementation detail, I would simply remove this sentence and leave the description above that defines what the user will achieve.